ql/src/test/results/clientpositive/llap/acid_bloom_filter_orc_file_dump.q.out

```diff
@@ -76,7 +76,7 @@ PREHOOK: Input: default@bloomtest
 #### A masked pattern was here ####
 -- BEGIN ORC FILE DUMP --
 #### A masked pattern was here ####
-File Version: 0.12 with ORC_517
+File Version: 0.12 with ORC_14
```

Comment:
Is this change expected?

Reply:
Yes, ORC_14 is the latest Writer version (for 1.6) as per:

https://github.com/apache/orc/blob/a1671f1e8abf748178c07e2a03b03f00268c8863/java/core/src/java/org/apache/orc/OrcFile.java#L177